subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/bintray/ShipkitBintrayPlugin.java

```diff
@@ -55,6 +55,16 @@ public void apply(final Project project) {
             LOG.lifecycle(welcomeMessage);
         });
 
+        bintrayUpload.doLast(task -> {
+                BintrayUploadTask bintrayUploadTask = (BintrayUploadTask) task;
+                if ((bintrayUploadTask.getFileUploads() == null || bintrayUploadTask.getFileUploads().length == 0) &&
```

Comment:
How about we use "doFrist" and fail if no artifacts to upload are configured? The exception message should tell the user to either:
 - disable bintrayUpload task / avoid running it
 - or configure the task so that it has artifacts to publish

It would be great to drop a ticket to Bintray plugin on GitHub, report this problem and offer PR to their code!

Reply:
It looks like we can't use `doFirst` because the files for upload are resolved during task execution.
Ad open a ticket to Bintray: Yes, I will do so, thx!